src/components/HowItWorksDialog.vue

```diff
@@ -63,16 +65,25 @@
         class="text-surface-2"
         @click="close"
       >
-        Okay
+        {{ t('home.howItWorks.okayButton') }}
       </v-btn>
     </template>
   </base-dialog>
 </template>
 
 <script setup>
+import { useI18n } from 'vue-i18n';
 import { ref } from 'vue';
+import { computed } from 'vue';
 import BaseDialog from '@/components/Global/BaseDialog.vue';
 
+const { t } = useI18n();
+
+const titleItems = computed(() => {
+  return [{ text: t('home.howItWorks.title')}];
+});
+
```

Comment:
I'm not sure we need this. Or the key or v-for in the title I think we can simply do `:title=t('home.howItWorks.title')"`